packages/account/src/Components/additional-kyc-info-modal/additional-kyc-info-form.tsx

```diff
@@ -0,0 +1,240 @@
+import React from 'react';
+import {
+    Autocomplete,
+    Button,
+    DesktopWrapper,
+    Loading,
+    MobileWrapper,
+    Modal,
+    SelectNative,
+    Text,
+} from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import { Localize } from '@deriv/translations';
+import classNames from 'classnames';
+import { isMobile } from '@deriv/shared';
+import { Field, FieldProps, Form, Formik, FormikErrors } from 'formik';
+import FormFieldInfo from '../form-field-info';
+import { FormInputField } from '../forms/form-fields';
+import { TListItem, getFormConfig } from './form-config';
+import { useSettings } from '../../../../api/src/hooks';
+
+const FormTitle = () => (
+    <Text
+        as='p'
+        size='s'
+        line_height='xxl'
+        align={isMobile() ? 'left' : 'center'}
+        className='additional-kyc-info-modal__form--header'
+    >
+        <Localize i18n_default_text='Please take a moment to update your information now.' />
+    </Text>
+);
+
+export const AdditionalKycInfoForm = observer(
+    ({ setError }: { setError: React.Dispatch<React.SetStateAction<string>> }) => {
+        const { client, ui, notifications } = useStore();
+        const { account_settings, residence_list } = client;
```

Comment:
Can we get these values from `useSettings` hook which gets `get_settings` api.  I believe we account_settings gets these values from `get_settings` as well. Correct me if im wrong 👀

Reply:
I believe we can use that as we're using this hook to submit the data. Thanks for the suggestion

Reply:
@thisyahlen-deriv I checked and we're not able to fill the initial values for formik using the data from useSettings. 
maybe we can investigate and see how we can handle this if it's okay?